src/Kestrel.Core/Kestrel.Core.csproj

```diff
@@ -4,7 +4,7 @@
     <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Core</AssemblyName>
     <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Core</RootNamespace>
     <Description>Core components of ASP.NET Core Kestrel cross-platform web server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
How was this working before without the 2.1 target?

Reply:
It wasn't. See the PR summary.